### PR TITLE
enable fee during request_delegate and begin_task

### PR DESCRIPTION
**Pull request by RainFallsSilent**

### CODE REVIEW

File: pallets/abc-demo/src/delegate.rs [hunk 1]
```diff
@@ -28,7 +28,18 @@ pub struct DelegateInfo {
     pub key3_rsa_pub_key: String,
 }
 
-pub fn request_single_delegate(account: AccountId32) {
+pub fn get_url(net_address: &NetAddress) -> String {
```
Comment (raindust) [inline pallets/abc-demo/src/delegate.rs]: It's better to return with Result<String>

File: pallets/abc-demo/src/delegate.rs [hunk 2]
```diff
@@ -40,7 +51,7 @@ pub fn request_single_delegate(account: AccountId32) {
             let content = base64::encode(buf);
             let request_url = format!(
                 "{}{}?content={}",
-                SERVICE_BASE_URL, APPLY_DELEGATE, &content
+                get_url(net_address), APPLY_DELEGATE, &content
```
Comment (raindust) [inline pallets/abc-demo/src/delegate.rs]: If parse net address error, we should abort this request

Comment (RainFallsSilent) [reply pallets/abc-demo/src/delegate.rs]: fixed

File: pallets/abc-demo/src/delegate.rs (hunk 1 above)
Comment (RainFallsSilent) [reply pallets/abc-demo/src/delegate.rs]: fixed

**Pull request merged**